sdk/communication/Azure.Communication.Rooms/src/Models/ParticipantProperties.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Communication.Rooms
+{
+    /// <summary> The ParticipantProperties. </summary>
+    [CodeGenModel("ParticipantProperties")]
```

Comment:
[nit] Why do you need the attribute here? Is this done to make the model internal?

Reply:
Yes exactly

Reply:
@m-nash is this a supported scenario? If not, could you please share the recommendation from the DPG team here?

Reply:
Looking at the autorest.md setting the flag generation1-convenience-client: true is set, so I believe this is supported. But will wait for your ack @m-nash

Reply:
This follows the patterns we have today.